### PR TITLE
Homework11: Pavel Mitrafanau

### PR DESCRIPTION
**Pull request by Pavel2812mi**
Good day, Sergei
My homework11 file is ready for review

### CODE REVIEW

File: Homework11_1.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        """checks if the book is taken"""
+        return self in Book.taken_books
+
+
+book1 = Book("War_of_the_worlds", "Wells", 290, True, "False")
+book2 = Book("Time_machine", "Wells", 314, True, "False")
+book3 = Book("Island of doctor Morrow", "Wells", 325, True, "False")
+
+
+class reader:
```
Comment (Stml89) [inline Homework11_1.py]: Class name starts with upper letter: `Reader`

File: Homework11_1.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    """A class to represent all library readers"""
+
+    def __init__(self, name, resereved_books, taken_books):
+        self.name = name
+        self.resereved_books = resereved_books
+        self.taken_books = taken_books
```
Comment (Stml89) [inline Homework11_1.py]:
```suggestion
        self.resereved_books = []
        self.taken_books = []
```

File: Homework11_1.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
+"""Библиотека"""
+
+
+class Book:
+    """A class to represent all available books in the library"""
+    reserved_books: list["Book"] = []
```
Comment (Stml89) [inline Homework11_1.py]:
```suggestion
from typing import List

...
reserved_books: List[str] = []
```

File: Homework11_2.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                                     * (1 + deposit.percentage_per_annum / 100)
+                                     ** deposit.deposit_term)
+            print(f"Итоговая сумма депозита "
+                  f"{deposit_name}: {end_deposit_amount}")
+        else:
+            print(f"Депозит с именем {deposit_name} не найден.")
+
+
+client1 = Bank("Pavel_Danilov", True)
+client1.create_deposit("Deposit_1", 1000, 2, 10)
```
Comment (Stml89) [inline Homework11_2.py]: if a user puts 1000 for 1 year, with 10% -> he should get ~1104.** after 1 year

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Pavel2812mi** [comment]
The requested changes have been made to the code for Homework11_1 and Homework11_2.

**Review by Stml89 — APPROVED** [review]

**Pull request merged**